src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -117,6 +122,17 @@ def boot_partition_needed?
         false
       end
 
+      def too_small_boot?
+        # for other partitions it is not needed as packager check disk usage, but /boot is special
+        # as it contain initrd that is generated and also bootloader code.
+        filesystem = devicegraph.filesystems.find { |f| f.mount_point == "/boot" }
```

Comment:
this should use `#mount_path` (which returns a string) instead of `#mount_point` which returns a `MountPoint` object.